integration_tests/features/TransactionInfo.feature

```diff
@@ -0,0 +1,26 @@
+@merge-mining
+    Feature: Transaction Info
+
+    @critical
+        Scenario: Get Transaction Info
+            Given I have a seed node NODE
+            And I have 1 base nodes connected to all seed nodes
```

Comment:
This base node doesn't seem to add any value

Reply:
The reason I added another base node was it doesn't seem if the network play well with only on base node. Will test again and revert. If it is not needed I will remove it.

Reply:
Tested - currently, this makes a difference in the time of the test, 1 base node the test runs for an hour, two base nodes it runs for 10 minutes.